homeassistant/components/whirlpool/sensor.py

```diff
@@ -106,15 +104,15 @@ class WhirlpoolSensorEntityDescription(
     WhirlpoolSensorEntityDescription(
         key="state",
         name="State",
-        icon=ICON_W,
-        has_entity_name=True,
+        translation_key="whirlpool_machine",
+        device_class=SensorDeviceClass.ENUM,
```

Comment:
Please set the options attribute too:

https://developers.home-assistant.io/docs/core/entity/sensor#properties

Reply:
It's funny you ask for this - I was working on it, but had some difficulties getting the list "right" (two different types).   Then thought "since you didn't ask, let's not do more in this PR".

The options definitely make automations 1000% easier!

This should do it.